Add new tile features (#28720)
home-assistant/home-assistant.github.io: source/_dashboards/tile.markdown

  type: list
{% endconfiguration %}

### Climate HVAC modes

> Widget that display buttons to control change the HVAC mode for a [climate](/integrations/climate).

Reviewer: ```suggestion
Widget that displays buttons to control the HVAC mode for a [climate](/integrations/climate).
```

<p class='img'>

>   <img src='/images/dashboards/tile-features/climate_hvac_modes.png' alt='Screenshot of the tile card with climate HVAC modes feature'>

Reviewer: ```suggestion
  <img src='/images/dashboards/tile-features/climate_hvac_modes.png' alt='Screenshot of the tile card with the climate HVAC modes feature'>
```

>   Screenshot of the tile card with climate HVAC modes feature

Reviewer: ```suggestion
  Screenshot of the tile card with the climate HVAC modes feature
```

</p>

```yaml
features:
  - type: "climate-hvac-modes"
    hvac_modes:
      - auto
      - heat_cool
      - heat
      - cool
      - dry
      - fan_only
      - "off"
```

{% configuration %}
type:
  required: true
  description: "`climate-hvac-modes`"
  type: string
hvac_modes:
  required: true
  description: List of modes to show on the card. The list can contain `auto`, `heat_cool`, `heat`, `cool`, `dry`, `fan_only` and `off`.
  type: list
{% endconfiguration %}

### Cover open/close

> Widget that display buttons to open, close or stop a [cover](/integrations/cover).

Reviewer: ```suggestion
Widget that displays buttons to open, close, or stop a [cover](/integrations/cover).
```

  type: string
{% endconfiguration %}

### Cover position

> Widget that display a slider to control the position for a [cover](/integrations/cover).

Reviewer: ```suggestion
Widget that displays a slider to control the position for a [cover](/integrations/cover).
```

<p class='img'>

>   <img src='/images/dashboards/tile-features/cover_position.png' alt='Screenshot of the tile card with cover position feature'>

Reviewer: ```suggestion
  <img src='/images/dashboards/tile-features/cover_position.png' alt='Screenshot of the tile card with the cover position feature'>
```

>   Screenshot of the tile card with cover position feature

Reviewer: ```suggestion
  Screenshot of the tile card with the cover position feature
```

</p>

```yaml
features:
  - type: "cover-position"
```

{% configuration %}
type:
  required: true
  description: "`cover-position`"
  type: string
{% endconfiguration %}

### Cover tilt

> Widget that display buttons to open, close or stop a [cover](/integrations/cover).

Reviewer: ```suggestion
Widget that displays buttons to open, close, or stop a [cover](/integrations/cover).
```

  type: string
{% endconfiguration %}

### Cover tilt position

> Widget that display a slider to control the tilt position for a [cover](/integrations/cover).

Reviewer: ```suggestion
Widget that displays a slider to control the tilt position for a [cover](/integrations/cover).
```

<p class='img'>

>   <img src='/images/dashboards/tile-features/cover_tilt_position.png' alt='Screenshot of the tile card with cover tilt position feature'>

Reviewer: ```suggestion
  <img src='/images/dashboards/tile-features/cover_tilt_position.png' alt='Screenshot of the tile card with the cover tilt position feature'>
```

>   Screenshot of the tile card with cover tilt position feature

Reviewer: ```suggestion
  Screenshot of the tile card with the cover tilt position feature
```

</p>

```yaml
features:
  - type: "cover-tilt-position"
```

{% configuration %}
type:
  required: true
  description: "`cover-tilt-position`"
  type: string
{% endconfiguration %}

### Fan speed

> Widget that display speed controls for a [fan](/integrations/fan).

Reviewer: ```suggestion
Widget that displays speed controls for a [fan](/integrations/fan).
```

  type: string
{% endconfiguration %}

### Lawn mower commands

> Widget that display buttons to control a [lawn mower](/integrations/lawn_mower).

Reviewer: ```suggestion
Widget that displays buttons to control a [lawn mower](/integrations/lawn_mower).
```

<p class='img'>

>   <img src='/images/dashboards/tile-features/lawn_mower_commands.png' alt='Screenshot of the tile card with lawn mower commands feature'>

Reviewer: ```suggestion
  <img src='/images/dashboards/tile-features/lawn_mower_commands.png' alt='Screenshot of the tile card with the lawn mower commands feature'>
```

>   Screenshot of the tile card with lawn mower commands feature

Reviewer: ```suggestion
  Screenshot of the tile card with the lawn mower commands feature
```

</p>

```yaml
features:
  - type: "lawn-mower-commands"
    commands:
      - start_pause
      - dock
```

{% configuration %}
type:
  required: true
  description: "`lawn-mower-commands`"
  type: string
commands:
  required: true
  description: List of commands to show on the card. The list can contain `start_pause` and `dock`.
  type: list
{% endconfiguration %}

### Light brightness

> Widget that display a slider to select the brightness for a [light](/integrations/light).

Reviewer: ```suggestion
Widget that displays a slider to select the brightness for a [light](/integrations/light).
```

  type: string
{% endconfiguration %}

### Light color temp

> Widget that display a slider to select the color temperature for a [light](/integrations/light).

Reviewer: ```suggestion
Widget that displays a slider to select the color temperature for a [light](/integrations/light).
```

<p class='img'>

>   <img src='/images/dashboards/tile-features/light_color_temp.png' alt='Screenshot of the tile card with light color temperature feature'>

Reviewer: ```suggestion
  <img src='/images/dashboards/tile-features/light_color_temp.png' alt='Screenshot of the tile card with the light color temperature feature'>
```

>   Screenshot of the tile card with light color temperature feature

Reviewer: ```suggestion
  Screenshot of the tile card with the light color temperature feature
```

</p>

```yaml
features:
  - type: "light-color-temp"
```

{% configuration %}
type:
  required: true
  description: "`light-color-temp`"
  type: string
{% endconfiguration %}

### Target temperature

> Widget that display buttons to select the target temperature for a [climate](/integrations/climate) or a [water heater](/integrations/water_heater).

Reviewer: ```suggestion
Widget that displays buttons to select the target temperature for a [climate](/integrations/climate) or a [water heater](/integrations/water_heater).
```

<p class='img'>

>   <img src='/images/dashboards/tile-features/target_temperature.png' alt='Screenshot of the tile card with target temperature feature'>

Reviewer: ```suggestion
  <img src='/images/dashboards/tile-features/target_temperature.png' alt='Screenshot of the tile card with the target temperature feature'>
```

>   Screenshot of the tile card with target temperature feature

Reviewer: ```suggestion
  Screenshot of the tile card with the target temperature feature
```

</p>

```yaml
features:
  - type: "target-temperature"
```

{% configuration %}
type:
  required: true
  description: "`target-temperature`"
  type: string
{% endconfiguration %}

### Vacuum commands

> Widget that display buttons to control a [vacuum](/integrations/vacuum).

Reviewer: ```suggestion
Widget that displays buttons to control a [vacuum](/integrations/vacuum).
```

  description: List of commands to show on the card. The list can contain `start_pause`, `stop`, `clean_spot`, `locate` and `return_home`.
  type: list
{% endconfiguration %}

### Water heater operation modes

> Widget that display buttons to control change the operation mode of a [water heater](/integrations/water_heater).

Reviewer: ```suggestion
Widget that displays buttons to control the operation mode of a [water heater](/integrations/water_heater).
```

<p class='img'>

>   <img src='/images/dashboards/tile-features/water_heater_operation_modes.png' alt='Screenshot of the tile card with water heater operation modes feature'>

Reviewer: ```suggestion
  <img src='/images/dashboards/tile-features/water_heater_operation_modes.png' alt='Screenshot of the tile card with the water heater operation modes feature'>
```

>   Screenshot of the tile card with water heater operation modes feature

Reviewer: ```suggestion
  Screenshot of the tile card with the water heater operation modes feature
```
